src/workerd/api/node/tests/zlib-nodejs-test.js

```diff
@@ -213,3 +214,117 @@ export const crc32Test = {
     });
   }
 }
+
+export const constantsTest = {
+  test() {
+    deepStrictEqual(JSON.stringify(constants, null, 2),`{
+  "Z_NO_FLUSH": 0,
+  "Z_PARTIAL_FLUSH": 1,
+  "Z_SYNC_FLUSH": 2,
+  "Z_FULL_FLUSH": 3,
+  "Z_FINISH": 4,
+  "Z_BLOCK": 5,
+  "Z_OK": 0,
+  "Z_STREAM_END": 1,
+  "Z_NEED_DICT": 2,
+  "Z_ERRNO": -1,
+  "Z_STREAM_ERROR": -2,
+  "Z_DATA_ERROR": -3,
+  "Z_MEM_ERROR": -4,
+  "Z_BUF_ERROR": -5,
+  "Z_VERSION_ERROR": -6,
+  "Z_NO_COMPRESSION": 0,
+  "Z_BEST_SPEED": 1,
+  "Z_BEST_COMPRESSION": 9,
+  "Z_DEFAULT_COMPRESSION": -1,
+  "Z_FILTERED": 1,
+  "Z_HUFFMAN_ONLY": 2,
+  "Z_RLE": 3,
+  "Z_FIXED": 4,
+  "Z_DEFAULT_STRATEGY": 0,
+  "ZLIB_VERNUM": 4880,
```

Comment:
Since we know there's a version mismatch between workerd and the internal repo, I would likely just remove the check for `ZLIB_VERNUM` in the test for now.

Reply:
We can just check for Object.keys().sort() strict equality?

Reply:
I'll do this for now, but we should fix our dependency mess when we can.